pipelines/internal/commands/run/run.go

```diff
@@ -494,12 +495,21 @@ func parse(line string) (*genomics.Action, error) {
 	var action genomics.Action
 
 	options := make(map[string]string)
+	flags := map[string]*bool{
+		"IGNORE_EXIT_STATUS":             &action.IgnoreExitStatus,
+		"RUN_IN_BACKGROUND":              &action.RunInBackground,
+		"ALWAYS_RUN":                     &action.AlwaysRun,
+		"ENABLE_FUSE":                    &action.EnableFuse,
+		"PUBLISH_EXPOSED_PORTS":          &action.PublishExposedPorts,
+		"DISABLE_IMAGE_PREFETCH":         &action.DisableImagePrefetch,
+		"DISABLE_STANDARD_ERROR_CAPTURE": &action.DisableStandardErrorCapture,
+	}
 	if n := strings.Index(line, "#"); n >= 0 {
 		for _, option := range strings.Fields(strings.TrimSpace(line[n+1:])) {
 			if n := strings.Index(option, "="); n >= 0 {
 				options[option[:n]] = option[n+1:]
-			} else {
-				action.Flags = append(action.Flags, strings.ToUpper(option))
+			} else if p, ok := flags[strings.ToUpper(option)]; ok {
+				*p = true
 			}
```

Comment:
Add an else case that returns an error so incorrect flags aren't silently ignored.

Reply:
Done